chromium/pages/popup/onion-rules.js

```diff
@@ -0,0 +1,38 @@
+function e(id) {
+return document.getElementById(id);
+}
+  
+  /**
+ * Fill in content into the popup on load
+ */
+document.addEventListener("DOMContentLoaded", function () {
+  // auto-translate all elements with i18n attributes
+  var elem = document.querySelectorAll("[i18n]");
+  for (var i=0; i < elem.length; i++) {
+    elem[i].innerHTML = chrome.i18n.getMessage(elem[i].getAttribute("i18n"));
+  }
+
+  showRules();
+});
+
+function _createTD(text) {
+  var td = document.createElement('td');
+  var textNode = document.createTextNode(text);
+  td.appendChild(textNode);
+  return td;
```

Comment:
There is a simpler way to do that:

```suggestion
  var td = document.createElement('td');
  td.textContent = text;
  return td;
```